setup.cfg

```
@@ -0,0 +1,49 @@
+[metadata]
+name = ukrdict
+url = https://github.com/serhii73/ukrdict
+project_urls =
+    CI: Travis = https://travis-ci.com/serhii73/ukrdict
+    GitHub: repo = https://github.com/serhii73/ukrdict
+    GitHub: issues = https://github.com/serhii73/ukrdict/issues
+author = serhii73
+author_email = aserhii@protonmail.com
+description = Library for searching ukrainian words meaning.
+long_description = file: README.rst
+license = MIT
+license_file = LICENSE.rst
+keywords =
+    Ukraine
+    Ukrainian
+    dictionary
+    tool
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Software Development :: Libraries
+    Topic :: Software Development :: Testing
+
+[options]
+python_requires = >=3.6
+include_package_data = True
+packages = find:
+packages_dir =
+    = .
+install_requires =
+    beautifulsoup4>=4.7
+    click>=7.0
+    lxml>=4.3
+    requests>=2.22
+
+[options.entry_points]
+console_scripts =
+    ukrdict = ukrdict.main:main
```

Pro tip: if you just rename that module into dunder-main (the file itself too, of course), you'd be able to invoke it via runpy as in `python -m ukrdict`:
```suggestion
    ukrdict = ukrdict.__main__:main
```